### ISSUE TITLE
Alt key icon is incorrect on Linux

### ISSUE THREAD

**Issue by KorigamiK** [zed-industries/zed#21938]
Edited by @mikayla-maki:

Note that the `alt` icon uses the macOS version, not the unicode `⎇`

Further, this style of keybinding is inconsistent with the Linux keybinding conventions used elsewhere in the app, which would show this as `Ctrl + Alt + B`.

![image](https://github.com/user-attachments/assets/16b45cc1-0e6c-44ee-846f-fd958e387f6e)

**Comment by mikayla-maki** [comment]
Aren't those the symbols for `ctrl-alt-b`? I don't see the problem here.

**Comment by KorigamiK** [comment]
> Aren't those the symbols for `ctrl-alt-b`? I don't see the problem here.

No, I meant to say, those are the symbols for `cmd` which is Mac specific. 
This should be platform specific, I think
https://github.com/zed-industries/zed/blob/d7eba54016c634ae931a245d9b377aac0f078ed1/crates/gpui/src/window.rs#L3078

**Comment by jansol** [comment]
^ is used on all platforms for `ctrl`, ⌘ is Mac-specific for `cmd`. Apparently ⎇ is the unicode symbol for `alt` (the one in the screenshot looks like the macOS `option` icon).

That said elsewhere Zed is showing the textual names of the keys on Linux.

**Comment by mikayla-maki** [comment]
Ahh, so the issue is that the `alt` key icon is the macos specific one, rather than the standardized `⎇`

**Comment by KorigamiK** [comment]
> Ahh, so the issue is that the `alt` key icon is the macos specific one, rather than the standardized `⎇`
I'd say not using the unicode symbol fir linux and just using `alt` as text would be fine as that's what other ui elements have.

**Comment by iamnbutler** [comment]
We should just case this to use the proper alt icon for platforms not macOS. This should be a simple fix and would be a great first issue for someone looking to contribute!

**Comment by ConradIrwin** [comment]
Linux should now just show the text of modifiers not the icons.
